test/dlc_tests/container_tests/bin/benchmark/run_mxnet_inference_performance_cpu

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+TIMESTAMP=$(date "+%Y-%m-%d-%H-%M-%S")
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs/cpu
+LOG_FILE=inference_results_${COMMIT_INFO}_${TIMESTAMP}.txt
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+set -e
+pip install awscli
+echo Fetch s3 bucket >&2
+aws s3 sync s3://aws-ml-platform-datasets/imagenet/pass-through/ /home/ubuntu/data
+echo Fetch finished >&2
+git clone https://github.com/awslabs/deeplearning-benchmark.git ${HOME_DIR}/artifacts/mxnet/deeplearning-benchmark  && cd ${HOME_DIR}/artifacts/mxnet/
+START=$(date +%s)
+pip install psutil gluoncv==0.2.0b20180625 && export KMP_AFFINITY='granularity=fine,compact,1,0' && export OMP_NUM_THREADS=36 && python deeplearning-benchmark/image_classification/infer_imagenet.py \
```

Comment:
Please set up a plan for the future to update the gluoncv version being used here.

Reply:
Thanks for this nice catch. Try to remove the exact version number of testing dependencies